manim/mobject/types/opengl_vectorized_mobject.py

```diff
@@ -163,6 +172,14 @@ def set_stroke(
         background=None,
         recurse=True,
     ):
+        if color is not None:
+            self.stroke_color = Color(color)
+        if opacity is not None:
+            self.stroke_opacity = opacity
+        if recurse:
+            for submobject in self.submobjects:
+                submobject.set_stroke(color, width, opacity, background, recurse)
```

Comment:
Can you please pass these as keyword arguments?

Reply:
> Can you please pass these as keyword arguments?

Done and I forgot to move the test file into the opengl directory so did that too